qcodes/instrument_drivers/tektronix/Keithley_3706A.py

```diff
@@ -138,8 +127,35 @@ def close_channel(self, val: str) -> None:
         if val in forbidden_channels.split(','):
             warnings.warn('You are attempting to close channels that are '
                           'forbidden to close.', UserWarning, 2)
+
```

Comment:
It seems like the code below should perhaps be split out into a private method?

Reply:
done